meson.build

```diff
@@ -29,6 +29,26 @@ if api_bus == 'user'
    common_cflags += '-DUSE_USER_API_BUS'
 endif
 
+# Paths configuration according to used prefix
+path_conf = configuration_data()
+
+prefixdir = get_option('prefix')
+
+sysconfdir = get_option('sysconfdir')
+if not sysconfdir.startswith('/')
+    # sysconfdir is relative to the prefix
+    sysconfdir = prefixdir / get_option('sysconfdir')
+endif
+path_conf.set_quoted('SYSCONF_DIR', sysconfdir)
+
+path_h = configure_file(
+        output : 'path.h',
+        configuration : path_conf)
+
+# For lint to work properly the same include needs to be added manually to clang-tidy arguments
+add_project_arguments('-include', 'path.h', language : 'c')
```

Comment:
'config.h' is the more traditional name, in case we need to set other optional configuration defines.

Reply:
I know, but we already have `libhirte/common/config.h`, so wouldn't it be confusing to have 2 different `config.h` files in the project?